Homeworks/homework_6/airports/large_airport.py

```diff
@@ -0,0 +1,24 @@
+from Homeworks.homework_6.airports.airport import Airport
+from Homeworks.homework_6.aircrafts.helicopter import Helicopter
+
+
+class LargeAirport(Airport):
```

Comment:
you can create another parent class -- PassengerAirport to combine functionality of all passengers airports